tests-js/loan-settings/LoanSettingsEvents.js

```diff
@@ -0,0 +1,134 @@
+		underlyingToken = await TestToken.new(name, symbol, 18, TOTAL_SUPPLY);
+		testWrbtc = await TestWrbtc.new();
+
+		const sovrynproxy = await sovrynProtocol.new();
+		sovryn = await ISovryn.at(sovrynproxy.address);
+
+		await sovryn.replaceContract((await LoanClosings.new()).address);
+		await sovryn.replaceContract((await ProtocolSettings.new()).address);
+		await sovryn.replaceContract((await LoanSettings.new()).address);
+		await sovryn.replaceContract((await LoanMaintenance.new()).address);
+		await sovryn.replaceContract((await SwapsExternal.new()).address);
+		await sovryn.replaceContract((await LoanOpenings.new()).address);
+
+		await sovryn.setWrbtcToken(testWrbtc.address);
+
+		feeds = await PriceFeedsLocal.new(testWrbtc.address, sovryn.address);
+		await feeds.setRates(underlyingToken.address, testWrbtc.address, ether("0.01"));
+		const swaps = await SwapsImplLocal.new();
+		const sovrynSwapSimulator = await TestSovrynSwap.new(feeds.address);
+		await sovryn.setSovrynSwapContractRegistryAddress(sovrynSwapSimulator.address);
+		await sovryn.setSupportedTokens([underlyingToken.address, testWrbtc.address], [true, true]);
+		await sovryn.setPriceFeedContract(
+			feeds.address //priceFeeds
+		);
+		await sovryn.setSwapsImplContract(
+			swaps.address // swapsImpl
+		);
+		await sovryn.setFeesController(lender);
+
+		loanTokenLogicWrbtc = await LoanTokenLogicWrbtc.new();
+		loanToken = await LoanToken.new(lender, loanTokenLogicWrbtc.address, sovryn.address, testWrbtc.address);
+		await loanToken.initialize(testWrbtc.address, "iWRBTC", "iWRBTC"); //iToken
+		loanToken = await LoanTokenLogicWrbtc.at(loanToken.address);
+
+		const loanTokenAddress = await loanToken.loanTokenAddress();
+		if (lender == (await sovryn.owner())) await sovryn.setLoanPool([loanToken.address], [loanTokenAddress]);
+
+		await testWrbtc.mint(sovryn.address, ether("500"));
+
+        loanParams = {
+            "id": "0x0000000000000000000000000000000000000000000000000000000000000000",
+            "active": false,
+            "owner": constants.ZERO_ADDRESS,
+            "loanToken": underlyingToken.address,
+            "collateralToken": testWrbtc.address,
+            "minInitialMargin": ether("50"),
+            "maintenanceMargin": ether("15"),
+            "maxLoanTerm": "2419200"
+        };
+   	});
+
+    
+	describe("test LoanSettingsEvents", async () => {
+		it("test setupLoanParamsEvents", async () => {
+            tx = await sovryn.setupLoanParams([Object.values(loanParams)]);
+
+            await expectEvent(tx, "LoanParamsIdSetup", {owner: lender});
+
+            await expectEvent(tx, "LoanParamsSetup", 
+                {
+                    owner: lender, 
+                    loanToken: underlyingToken.address, 
+                    collateralToken: testWrbtc.address, 
+                    minInitialMargin: ether("50"), 
+                    maintenanceMargin: ether("15"), 
+                    maxLoanTerm: "2419200"
+                }
```

Comment:
in brownie there is one more param: `"id": "0x0"`
do we need to add it here as well?